Controller/CRUDController.php

```diff
@@ -265,6 +265,9 @@ public function editAction($id = null)
 
             // persist if the form was valid and if in preview mode the preview was approved
             if ($isFormValid && (!$this->isInPreviewMode() || $this->isPreviewApproved())) {
+                $object = $form->getData();
+                $this->admin->setSubject($object);
```

Comment:
Can you rename variables though? There will be less confusion. Maybe `$existingObject` and `$modifiedObject` for `editAction`, and `$defaultObject` / `$createdObject` for `createAction`, what do you think?